caas/kubernetes/provider/bootstrap_test.go

```diff
@@ -309,7 +309,7 @@ func (s *bootstrapSuite) TestBootstrap(c *gc.C) {
 		{
 			Name:            "mongodb",
 			ImagePullPolicy: core.PullIfNotPresent,
-			Image:           "mongo:3.6.6",
+			Image:           s.pcfg.GetJujuDbOCIImagePath(),
```

Comment:
do we want to hardcode the version here in test?
And manual update here if we want to upgrade version?